contributors/devel/development.md

```diff
@@ -286,7 +286,9 @@ different versions of Kubernetes.
 | 1.21 - 1.22    | 1.16.7      |
 | 1.23           | 1.17        |
 | 1.24           | 1.18        |
-| 1.25+          | 1.19        |
+| 1.25           | 1.20.10     |
+| 1.26 - 1.29    | 1.21.7      |
+| 1.30           | 1.22        |
```

Comment:
If we are keeping specific patch versions, then this should be 1.22.1
https://github.com/kubernetes/kubernetes/blob/7ea3d0245a63fbbba698f1cb939831fe8143db3e/build/dependencies.yaml#L119